content/usgs-packages/Application_Part2.Rmd

```diff
@@ -7,12 +7,12 @@ image: "usgs-packages/static/img/workflow.svg"
 output: USGSmarkdowntemplates::hugoTraining
 parent: Introduction to USGS R Packages
 weight: 42
-draft: true
+draft: "FALSE"
```

Comment:
did you test if "FALSE" and FALSE (and false I think would work) all work?

Reply:
Used "FALSE" because of @wdwatkins's comment here: https://github.com/USGS-R/USGSmarkdowntemplates/pull/20. He notes `set it to "FALSE" (need to quote it!)`